ini/native.ini

```diff
@@ -34,14 +34,14 @@ src_filter      = ${common.default_src_filter} +<src/HAL/LINUX>
 [simulator_common]
 platform          = native
 framework         =
-build_flags       = ${common.build_flags} -std=gnu++17 -D__PLAT_NATIVE_SIM__ -DU8G_HAL_LINKS -I/usr/include/SDL2 -IMarlin -IMarlin/src/HAL/NATIVE_SIM/include -IMarlin/src/HAL/NATIVE_SIM/u8g
+build_flags       = ${common.build_flags} -std=gnu++17 -D__PLAT_NATIVE_SIM__ -DU8G_HAL_LINKS -I/usr/include/SDL2 -IMarlin -IMarlin/src/HAL/NATIVE_SIM/u8g
 src_build_flags   = -Wall -Wno-expansion-to-defined -Wcast-align
 release_flags     = -g0 -O3 -flto
 debug_build_flags = -fstack-protector-strong -g -g3 -ggdb
 lib_compat_mode   = off
 src_filter        = ${common.default_src_filter} +<src/HAL/NATIVE_SIM>
 lib_deps          = ${common.lib_deps}
-  MarlinSimUI=https://github.com/p3p/MarlinSimUI/archive/master.zip
+  MarlinSimUI=https://github.com/thinkyhead/MarlinSimUI/archive/updated_marlin_hal_2093.zip
```

Comment:
I can merge your changes again and publish a new version, then you could use a zip link going forward (I won't be maintaining backwards compatibility), but every commit to Marlin especially if it hits the HAL or PIN naming convention is likely to break compatibility, because the sim is just a development tool and intrinsically linked to internals I don't see a reason to stress about maintaining compatibility with anything other than current bugfix-2.0.x.

Reply:
> I can merge your changes again and publish a new version 

That sounds good. ZIP link is definitely the preferred way. Tagging version numbers –in any branch– also helps, but someone can always use the commit ID if some specific point in time is needed.

Reply:
using this link: `https://github.com/p3p/MarlinSimUI/archive/0.0.2.zip` should be fine for now